### PR TITLE
Fixes multiple file upload for django 3.2.19 resolving issue #1353

### PR DESCRIPTION
**Pull request by joshwizzy**
## What?
I've have modified the `clean` method of the `RestrictedFileFormField` class to correctly handle multiple file uploads
#1353.
## Why?
These changes in Django 3.2.19 broke the file upload widget. See #1353  for more information.
## How?
The modified `clean` function accounts for  the case `data` value is a list.

## Summary by CodeRabbit

- **Bug Fixes**
  - Enhanced the file validation process to handle multiple files, checking their content types and sizes more accurately.

### CODE REVIEW

**Comment by tylerecouture** [comment]
@coderabbitai review

File: src/utilities/fields.py [hunk 1]
```diff
@@ -225,24 +225,34 @@
...
-                    str(self.content_types)))
-        except AttributeError:
-            pass
-
-        return data
-
+            except AttributeError as e:
+                pass
+       
+        return result
```
[the remark this answers is not in the document]
Comment (joshwizzy) [reply src/utilities/fields.py]: @coderabbit review

[the remark this answers is not in the document]
Comment (tylerecouture) [reply src/utilities/fields.py]: @joshwizzy  you missed the "ai" at the end of the coderabitai user =)

[the remark this answers is not in the document]
Comment (joshwizzy) [reply src/utilities/fields.py]: @coderabbitai review

**Comment by tylerecouture** [comment]
This still doesn't appear to actually work in the app:

MultiUploadFail

**Pull request merged**